packages/client/components/ActivityLibrary/ActivityDetails/hooks/useActivityDetails.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import {ActivityDetailsQuery} from '~/__generated__/ActivityDetailsQuery.graphql'
+import {CategoryID, QUICK_START_CATEGORY_ID} from '../../Categories'
+import {MeetingTypeEnum} from '../../../../../server/postgres/types/Meeting'
+import {ActivityId, getActivityIllustration} from '../../ActivityIllustrations'
+import {useHistory} from 'react-router'
+
+export type NoTemplatesMeeting = Exclude<MeetingTypeEnum, 'retrospective' | 'poker'>
+
+export const MEETING_ID_TO_NAME: Record<NoTemplatesMeeting, string> = {
+  teamPrompt: 'Standup',
+  action: 'Team Check-In'
+}
+
+export const MEETING_TYPE_TO_CATEGORY_ID: Record<NoTemplatesMeeting, CategoryID> = {
+  action: 'standup',
+  teamPrompt: 'standup'
+}
+
+export const ACTIVITY_TYPE_DATA_LOOKUP: {
```

Comment:
+1 Let's flip the mappings here, so it's `MeetingTypeEnum -> field -> data` instead of `field -> MeetingTypeEnum -> data`